packages/docusaurus-theme-openapi-docs/src/markdown/utils.ts

```diff
@@ -33,11 +37,11 @@ export function guard<T>(
   return "";
 }
 
-export function render(children: Children): string {
+export function render(children: ReactNode): string {
```

Comment:
I didn't mark this function as deprecated since it is slightly different to the plugin's equivalent:

```diff
export function render(children: ReactNode): string {
  if (Array.isArray(children)) {
-     return children.filter((c) => c !== undefined).join("");
+     const filteredChildren = children.filter((c) => c !== undefined);
+     return filteredChildren
+       .map((i: any) => (Array.isArray(i) ? i.join("") : i))
+       .join("");
  }
  return `${children ?? ""}`;
}
```